CHANGELOG.md

```diff
@@ -1,4 +1,82 @@
-## [Unreleased][unreleased]
+# Table of Contents
+
+- [Planned](#planned)
+    - [0.12.0](#0120---tbd)
+- [Scheduled](#scheduled)
+    - [0.11.2](#0112---20171129)
+- [Released](#released)
+    - [0.11.1](#0111---20171024)
+    - [0.10.4](#0104---20171024)
+    - [0.11.0](#0110---20170816)
+    - [0.10.3](#0103---20170524)
+    - [0.10.2](#0102---20170501)
+    - [0.10.1](#0101---20170327)
+    - [0.10.0](#0100---20170307)
+    - [0.9.9 and prior](#099---20170202)
+
+# Planned
+
+This section describes planned releases of Kong and their thematic fomr a
+high-level perspective. Those releases do not have a fixed release date yet.
+
+## [0.12.0] - TBD
+
+This release will mainly focus around two new major features:
+
+- The introduction of **health checks** in the load-balancer
+- The decoupling of the API entity into **Routes** and **Services** for better
+  granularity when configuring plugins.
+
+[Back to TOC](#table-of-contents)
+
+# Scheduled
+
+This section describes upcoming releases that have a release date, along with
+a detailed changeset of their content.
+
+## [0.11.2] - 2017/11/29
+
+##### Core
+
+- Avoid logging some unharmful error messages related to clustering.
+  [#3002](https://github.com/Kong/kong/pull/3002)
+- Improve performance and memory footprint when parsing multipart request
+  bodies.
+  [Kong/lua-multipart#13](https://github.com/Kong/lua-multipart/pull/13)
+
+##### Admin API
+
+- PUT requests with payloads containing non-existing primary keys for entities
+  now return HTTP 404 Not Found, instead of HTTP 200 OK without a response
+  body.
+  [#3007](https://github.com/Kong/kong/pull/3007)
+- On the `/` endpoint, ensure `enabled_in_cluster` shows up as an empty JSON
+  Array (`[]`), instead of an empty JSON Object (`{}`).
+  Thanks [@hbadgi](https://github.com/hbagdi) for the patch!
+  [#2982](https://github.com/Kong/kong/issues/2982)
+
+##### Plugins
+
+- :fireworks: key-auth: New endpoints to manipulate API keys!
```

Comment:
Same here - can we link to Issues? And perhaps to messages on the Mailing List?

Reply:
Desfinitely an oversight, thanks for the catch. We always link to the PR, and give kudos to the contributor if they are external to the company. I added the kudos and the PR link (no issue for this).